jsonnet/kube-thanos/kube-thanos-compact.libsonnet

```diff
@@ -66,6 +68,10 @@ function(params) {
         '--retention.resolution-1h=' + tc.config.retentionResolution1h,
         '--delete-delay=' + tc.config.deleteDelay,
       ] + (
+        if tc.config.compactConcurrency != 1 then [ '--compact.concurrency=' + tc.config.compactConcurrency] else []
+      ) + (
+        if tc.config.downsampleConcurrency != 1 then ['--downsample.concurrency=' + tc.config.downsampleConcurrency] else []
+      ) + (
```

Comment:
What about skipping the equals 1 check and always explicitly setting the two flags?
This way users can always know the concurrency without knowing the default value is 1.

Reply:
I will get that done for you.